src/opera/pge/base/base_pge.py

```diff
@@ -346,7 +346,7 @@ def _core_filename(self, inter_filename=None):  # pylint: disable=unused-argumen
         """
         time_tag = get_time_for_filename(self.production_datetime)
 
-        return f"{self.PROJECT}_{self.LEVEL}_{self.NAME}_{time_tag}"
+        return f"{self.PROJECT}_{self.LEVEL}_{self.NAME}-{time_tag}"
```

Comment:
Hi @ramvikrams, thanks for putting this PR together. My only note is that the underscore to hyphen change does not need to be applied to the placeholder filename we apply in `base_pge.py`. If you could revert this change I would be happy to merge the rest of this branch.

Reply:
Done